day_number/task9.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "date"
+
+class RegularDay
+  def initialize(year, month, day)
+    @year = year
+    @month = month
+    @day = day
+  end
+
+  def date_to_number
+    "Day #{Date.new(@year, @month, @day).yday} of the year"
```

Comment:
you can play with date format 
https://apidock.com/ruby/DateTime/strftime

Reply:
made changes :)